ministryofjustice/laa-apply-for-criminal-legal-aid
# Crimap 45 eforms redirect

## Description of change
Adds a partial for the eForms redirect if NINO is not given.

As we are no longer checking for partner, we agreed that this can be NINO specific for now. Hence the `NinoExitController` etc

## Link to relevant ticket

## Notes for reviewer
- Please check copy looks okay
- Note: changes to NINO page where the question is replaced by single input with link for what is now "No" option will be added in different PR.
- Also "Back to you applications" just goes to '#' at the moment pending that page actually being created

## Screenshots of changes (if applicable)
[CRIMAP-45](https://dsdmoj.atlassian.net/jira/software/projects/CRIMAP/boards/897?selectedIssue=CRIMAP-45)

### After changes:
<img width="998" alt="Screenshot 2022-08-03 at 16 05 23" src="https://user-images.githubusercontent.com/13377553/182642450-1da76b17-d416-4998-9c01-f3da0a425340.png">

## How to manually test the feature
Cycle through application until you reach NINO page > click no > click "Save and Continue"

This should redirect to the new eForms partial as per the prototype
